lib/ui/painting.dart

```diff
@@ -825,7 +825,24 @@ enum BlendMode {
 
 /// Quality levels for image filters.
 ///
-/// See [Paint.filterQuality].
+/// When scaling up typically the quality is lowest at [none], higher at [low],
+/// even higher at [medium], and the highest at [high].
+///
+/// When scaling down, the higher the scale factor the less effective the [high]
+/// level is. Beyond certain scale level [medium] may provide better visual
+/// result than [high] (see example below).
+///
+/// When building for the web using the `--web-renderer=html` option, filter quality has
+/// no effect. All images are rendered using the respective browser's default setting.
+///
+/// Example:
+///
+/// ![](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/filter_quality.png)
```

Comment:
We should offer some alternative text that can take the place of (be an alternative to) the image for when people don't have images enabled. (Or, alternatively, rephrase the text to not refer to the example, such that it reads naturally with or without the image.) See https://html.spec.whatwg.org/multipage/images.html#alt

Reply:
I followed the other examples in our code and rephrased the text to not refer to the example.